web/res/php/webtemplate.php

```diff
@@ -27,28 +27,229 @@ public function head() {?>
   <title><?php echo $this->t;?></title>
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap-glyphicons.css">
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.3/css/bootstrap.min.css" integrity="sha384-MIwDKRSSImVFAZCVLtU0LMDdON6KVCrZHyVQQj6e8wIEJkW4tvwqXrbMIya1vriY" crossorigin="anonymous">
+  <!-- CUSTOM THEMES HERE!!! -->
```

Comment:
Note that this project uses Bootstrap 4.0.0 elements. That's why the buttons look broken with these themes.